src/requests/flow.ts

```diff
@@ -58,7 +62,7 @@ export async function getLatestFlowGraph(
   const response: { published_flows: { data: FlowGraph }[] } =
     await client.request(
       gql`
-        query GetLatestPublishedFlowData($flowId: uuid!) {
+        query GetLatestPublishedFlowData($flowId: string!) {
```

Comment:
Why has this changed?

Reply:
Sorry! Yes, this was a change I made to the wrong query, could pick up, I'll revert